server/lib/currency.ts

```diff
@@ -264,3 +265,20 @@ export function reduceArrayToCurrency(array: AmountWithCurrencyAndDate[], curren
     },
   );
 }
+
+/**
+ * Gets exchange rate between given crypto currency and fiat.
+ */
+export async function getCryptoFxRate(cryptoCurrency: string, fiatCurrency: string): Promise<string> {
+  if (!cryptoCurrency || !fiatCurrency) {
+    return null;
+  }
+  try {
+    const { rate: cryptoToUSDRate } = await getCryptoToUSDRate(cryptoCurrency);
+    const USDToCollectiveCurrencyRate = await getFxRate('USD', fiatCurrency);
+    return cryptoToUSDRate * USDToCollectiveCurrencyRate;
+  } catch (error) {
+    logger.error(error);
+    return null;
+  }
+}
```

Comment:
At line 144 of `server/paymentProviders/thegivingblock/index.js` we're fetching a crypto fxRate without the need of this. What is justifying it?